packages/library/src/commonMain/kotlin/io/realm/Ignore.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm
+
+@Retention(AnnotationRetention.SOURCE)
```

Comment:
In Java we use runtime retention due to other tools sometimes needing this information and use reflection to get to it. It does add a little extra to the binary size, but IMO the flexibility outweighs that. Not sure what you think?

https://github.com/realm/realm-java/blob/master/realm-annotations/src/main/java/io/realm/annotations/Ignore.java#L24

Reply:
As we cannot access the annotations on Kotlin native, we also need some other representation, so maybe we should just expose is part of our schema meta-data (including info on excluded fields 🤪).  

I don't have any strong preference whether to keep it at runtime for convenience as JVM users would probably expect it to be there.

Reply:
👌 moving it to Runtime.